client/app/components/manager-navbar/manager-navbar.module.js

```diff
@@ -11,79 +11,86 @@ angular.module("App").run((LANGUAGES, managerNavbar, User, constants, translator
 
         managerNavbar.setExternalLinks(universes);
 
-        managerNavbar.setInternalLinks([
-            {
-                label: translator.tr("common_menu_support_assistance"),
-                subLinks: [
-                    {
-                        label: translator.tr("common_menu_support_all_guides"),
-                        url: constants.urls[user.ovhSubsidiary].guides.home
-                    },
-                    {
-                        label: translator.tr("common_menu_support_new_ticket"),
+        const currentSubsidiaryURLs = constants.urls[user.ovhSubsidiary];
+
+        const subLinks = [{
+            label: translator.tr("common_menu_support_new_ticket"),
+            click: () => {
+                if (!OtrsPopupService.isLoaded()) {
+                    OtrsPopupService.init();
+                } else {
+                    OtrsPopupService.toggle();
+                }
+            }
+        }, {
+            label: translator.tr("common_menu_support_list_ticket"),
+            url: "#/support"
+        }];
+
+        if (currentSubsidiaryURLs != null) {
```

Comment:
!== ?

Reply:
`x != null` === `x != null && x != undefined`

Reply:
I understand @AxelPeter feedback.

What do you think about this syntax : `if(!_.isEmpty(currentSubsidiaryURLs)) ` ? :)

Reply:
I actually should have adopted this syntax to start with.

Reply:
Can you fix these PR plz ? :)